tests/unit/models/item-test.js

```diff
@@ -59,3 +59,102 @@ test('allPackageCategories', function(assert) {
 
   assert.equal(record.get('allPackageCategories.firstObject.id'), [pkgCategory.id]);
 });
+
+
+test('isItem', function(assert){
+  var record, pkgType;
+  var subject = this.subject();
+  assert.expect(1);
+
+  Ember.run(function() {
+    subject.store.createRecord('package_type', {id: 5, code: "ABC"});
```

Comment:
Can we do **pkgType = subject.store.createRecord('package_type', {id: 5, code: "ABC"});**?